### PR TITLE
Add testclf tool

### PR DESCRIPTION
**Pull request by michdolan**
Adds CLI tool to apply a Common LUT Format (CLF) transform to input RGB data to test its
accuracy. It supports transforming images and RGB values.

This can be linked to the CLF format document as a Python-only solution for verifying CLFs being contributed to the project.

### CODE REVIEW

**Comment by michdolan** [comment]
Note: The current CI failures will be resolved by the dependency version update in #48

File: utilities/tools/testclf/testclf.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+"""
+Test CLF
+========
+
+CLI tool to apply a Common LUT Format (CLF) transform to input RGB data
+to test its accuracy.
+
+This module can also be imported and the function ``test_clf`` called
+to use the tool programmatically.
+"""
```
Comment (doug-walker) [inline utilities/tools/testclf/testclf.py:12]: Given that there are CTF files in the repo too, you may want to mention that this tool works with formats other than CLF.  For example:

"In addition to CLF, this tool will work with any other LUT format supported by OpenColorIO, including CTF."

**Review by doug-walker — DISMISSED** [review]
Should this be here or under apps in OpenColorIO itself?  

(I know pip doesn't install the Python apps yet, but I think maybe Remi mentioned that he was thinking about adding that?)

File: utilities/tools/testclf/testclf.py (hunk 1 above)
Comment (michdolan) [reply utilities/tools/testclf/testclf.py:15]: Good point. Added the comment.

**Pull request merged**